packages/frontend/src/app/(sticky-header)/category/[[...path]]/page.tsx

```diff
@@ -247,7 +247,7 @@ export default async function Category({ params }: { params: { path: any } }) {
     posts = GetPostSummaries(targetCategory.relatedPosts)
     postsCount = targetCategory.relatedPostsCount
   } catch (err) {
-    console.error('Fetch posts failed!', err)
+    LogError(err)
```

Comment:
`LogError` only handles axios error. But in this `catch`, we will catch other errors.